MekHQ/data/universe/factions.xml

```diff
@@ -1110,6 +1112,8 @@ successor - unimplemented tag describing another faction code as the specified f
         <layeredForceIconLogoFilename>Federated Commonwealth.png</layeredForceIconLogoFilename>
         <colorRGB>248,212,44</colorRGB>
         <tags>is,super,playable</tags>
+        <start>3028</start>
+        <end>3068</end>
```

Comment:
This also could be replaced with - 24 April 3067, when things got officially dissolved.

Reply:
Here, I wasn't sure if it accepted LocalDate or just year; I couldn't spy any other uses of LocalDate, so my concern with using 3067 is that it would cause the faction to end on 01/01/3067. By using 3068 we ensure the faction still exists for the period of 3067 that the FedCon still existed.